src/utils/splitChildren.tsx

```diff
@@ -12,7 +12,8 @@ import { RangeSeries } from '../components/Series/RangeSeries';
 import { ScatterSeries } from '../components/Series/ScatterSeries';
 
 interface PlotChildren {
-  seriesAndAnnotations: ReactElement[];
+  series: ReactElement[];
+  annotations: ReactElement | null;
```

Comment:
annotations should be an array too. It's allowed to have multiple groups of annotations.

Reply:
okey